### PR TITLE
[hi] Localizing portability.md

### LINKED ISSUE #1166: [New term] `Portability` for `Hindi`
#### What this term is about? (briefly)
-  Software portability is the possibility to use the same software in different environments.

#### Why this term needs to be added? (briefly) 
- Localizing this term in Hindi will help Hindi folks learn about this term in their native language.

Link here: https://glossary.cncf.io/portability/

### PR DESCRIPTION
**Pull request by Omkar0114**
Signed-off-by: Omkar kulkarni <88308267+Omkar0114@users.noreply.github.com>
fixes #1166

### CODE REVIEW

File: content/hi/portability.md
Comment (anubha-v-ardhan) [inline content/hi/portability.md:7]:
```suggestion
एक सॉफ्टवेयर विशेषता, पोर्टेबिलिटी पुन: प्रयोज्यता (reusability) का एक रूप है जो कुछ ऑपरेटिंग वातावरण में "अवरुद्ध" (लॉक-इन) से बचने में मदद करती है, जैसे क्लाउड प्रदाता, ऑपरेटिंग सिस्टम(os) या विक्रेता (vendors)।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @Omkar0114

File: content/hi/portability.md
Comment (Omkar0114) [reply content/hi/portability.md]: Updated :)

**Comment by Omkar0114** [comment]
Hey, @anubha-v-ardhan I made suggested changes, You can check it out. 🙌

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!

Thank you @Omkar0114. Please wait for one more approval before we can merge this.

**Comment by Omkar0114** [comment]
Yes. Thanks a lot @anubha-v-ardhan 😃 & @Garima-Negi

**Comment by Omkar0114** [comment]
Hey, @sayantani11 & @Garima-Negi Can you review this PR, already one LGTM has been added. So I can work on another term 😃

File: content/hi/portability.md
Comment (Garima-Negi) [inline content/hi/portability.md:7]: we should capitalize (os) to (OS)

Comment (Garima-Negi) [reply content/hi/portability.md:7]: replace with `वातावरणों` to match plural `systems`

**Review by Garima-Negi — CHANGES_REQUESTED** [review]
few nits, rest looks good

File: content/hi/portability.md
Comment (Omkar0114) [reply content/hi/portability.md]: Updated :)

**Comment by Omkar0114** [comment]
Done with all requested changes @Garima-Negi check it out! 🙂

**Review by Garima-Negi — APPROVED** [review]

**Pull request merged**
